README.md

```diff
@@ -37,46 +37,35 @@ The prefered way to install HOMS is to use Docker
   ```
 3. For OS X users: make path to folder with HOMS shared in `Docker -> Preferences... -> File Sharing`.
 
-4. Copy your (or default) configs to `/etc/hydra/homs/`:
+4. Copy your (or default) `.env` file to your project's directory:
 
   ```
-  cp config/bpm.yml.sample /etc/hydra/homs/bpm.yml
-  cp config/database.yml.sample /etc/hydra/homs/database.yml
-  cp config/hbw.yml.sample /etc/hydra/homs/hbw.yml
-  cp config/homs_configuration.yml.sample /etc/hydra/homs/homs_configuration.yml
-  cp config/imprint.yml.sample /etc/hydra/homs/imprint.yml
-  cp config/sources.yml.sample /etc/hydra/homs/sources.yml
-  cp config/secrets.yml.sample /etc/hydra/homs/secrets.yml
-  ```
-
-5. Copy your (or default) `.env` file to your project's directory:
-
-  ```
-  cp homs.env.sample .env
+  cp .env.sample .env
   ```
+  
+   All variables are set in `.env` file. There you can change them, if you want to.
 
-6. Change [Minio](https://github.com/minio/minio) credentials in `.env` file.
+5. Change [Minio](https://github.com/minio/minio) credentials in `.env` file.
 
-7. Be sure to update secret key in `/etc/hydra/homs/secrets.yml`. You can generate key with this command:
+6. Be sure to update secret key in `/etc/hydra/homs/secrets.yml`. You can generate key with this command:
 
   ```
   openssl rand -hex 64
   ```
 
-8. Run `docker-compose`:
+7. Run `docker-compose`:
 
   ```
   docker-compose up -d
   ```
 
-9. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
-
-10. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
+8. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
 
-11. (Optional) If you want to use demo processes navigate to Camunda Admin and create user with `user@example.com` email.
+9. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
 
-12. Login at [HydraOMS](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+10. (Optional) If you want to use demo processes navigate to Camunda Admin and create user with `user@example.com` email.
```

Comment:
Step 9 is pretty much the same as this step.

Reply:
Yes, but it's optional. I decided to leave it as it was before, but change number.